contracts/WidoConfig.sol

```diff
@@ -51,4 +64,10 @@ contract WidoConfig is IWidoConfig, OwnableUpgradeable {
             }
         }
     }
+
+    function setBank(address bank)  external override {
```

Comment:
also, i think this should be "onlyOwner" ?

Reply:
I would also check that the bank address is not zero, just for paranoia ^_^

Reply:
Thank you for catching. Fixed.